### ISSUE TITLE
What is the maintenance status of this repository? 

### ISSUE THREAD

**Issue by Chef-Chungus** [NoahZinsmeister/web3-react#163]
@NoahZinsmeister - We're actively using web3-react in our project and would like to continue to use it. It looks like there are lots of other projects which rely on web3-react also. 

What's the current maintenance status? there are many PRs and issues open dating back a long time. 

If funding is an issue, perhaps we can assist.

**Comment by Nacho97kalvin** [comment]
I believe its not being maintained for at least a year, actually i think there are diferent packages for different propose (but it keeps calling with the same namel ), see the repo below:
https://github.com/NoahZinsmeister/web3-react
sadly for me that repo dont have a good documentation like the previous one

**Comment by Blackglade** [comment]
Came across this repo as well and had trouble getting it working. The documentation isn't much help. Will probably switch off to something else, this isn't worth it.

**Comment by pakokrew** [comment]
Upping this

This project is used in a lot of dapps, including Uniswap. Let find some funding and get it back on rails.

**Comment by mtsalenc** [comment]
There were some commits 4 days ago. I love web-react let's not let it die.

**Comment by ccarnino** [comment]
Someone could help with GitGrants or Optimism's Retroactive Funding. But @NoahZinsmeister what is the real problem? Funding or lack of time?

**Comment by outdoteth** [comment]
Yep, I see @NoahZinsmeister has made quite a few commits on `main` today as well, so looks like it is maintained actively.

**Comment by Chef-Cheems** [comment]
@NoahZinsmeister hi, Chef Cheems from Pancakeswap.
We wonder whether slow development here is because of luck of funding or lack of maintainers?
We can assist with funding. Possibly with maintaining a little bit.

We could fork it of course but we'd rather see this package flourish. ❤️

**Comment by ekenigs** [comment]
Any updates here?

**Comment by NoahZinsmeister** [comment]
hey all - sorry for the lack of responsiveness here. rest assured, web3-react will continue to be maintained going forward! however, i do have some important updates:

frankly, the v6 release had some significant warts, and the maintenance overhead was getting too high for me to handle. however, there's a v8 design in the `main` branch which i just released under `beta` tags that hopefully will solve a lot of those issues, and lead to a superior devex in the long run. i'm planning to much more actively maintain v8, with maybe some help from others :)

i know this upgrade may cause some short-term pain (as has the lack of maintenance thus far on v6), for which i apologize. in general, my recommendation going forward is to try to upgrade to v8 if at all possible. if not, v6 is of course still usable, though contributions will likely have to made via forks/local patches/etc.

see the readme in `main` for more details! https://github.com/NoahZinsmeister/web3-react/tree/main#web3-react-beta

**Comment by TrejGun** [comment]
@NoahZinsmeister thanks for the update. Just one question: are you going to bring back dropped connectors like trezor and ledger?

**Comment by NoahZinsmeister** [comment]
my thinking on hardware wallets is that browser/desktop extensions (think metamask, frame) are the more natural integrators. not fully opposed, though, and of course it's always possible to write a standalone connector

**Comment by Lucienest** [comment]
@NoahZinsmeister it'd be great if you support the hardware connectors since our dapps rely on them.
Additionally, V8 should give the same or more flexibility as V6,
such as the freedom to choose between Web3/Ethers.
